guides/overview-apps.md

```diff
@@ -1,15 +1,3 @@
-# Wix App APIs
+# Apps made by Wix
 
-Customize Wix Apps with Velo APIs - including Stores, Bookings, Events, and Pay - to add even more features and functionality to your site. Combine APIs to create a unique app experience for your users. 
-
-*   **[wix-stores](https://www.wix.com/velo/reference/wix-stores.html)**
-*   **[wix-stores-backend](https://www.wix.com/velo/reference/wix-stores-backend.html)**
-*   **[wix-pay](https://www.wix.com/velo/reference/wix-pay.html)**
-*   **[wix-pay-backend](https://www.wix.com/velo/reference/wix-pay-backend.html)**
-*   **[wix-events](https://www.wix.com/velo/reference/wix-events)**
-*   **[wix-events-backend](https://www.wix.com/velo/reference/wix-events-backend)**
-*   **[wix-bookings](https://www.wix.com/velo/reference/wix-bookings)**
-*   **[wix-bookings-backend](https://www.wix.com/velo/reference/wix-bookings-backend)**
-*   **[wix-paid-plans](https://www.wix.com/velo/reference/wix-paid-plans)**
-*   **[wix-paid-plans-backend](https://www.wix.com/velo/reference/wix-paid-plans-backend)**
-*   **[wix-pricing-plans-backend](https://www.wix.com/velo/reference/wix-pricing-plans-backend)**
\ No newline at end of file
+[Apps made by Wix](https://dev.wix.com/docs/develop-websites) offer a range of powerful business solutions that extend the capabilities of existing Wix products and introduce new functionalities. A site owner can install one or more of these solutions on their site and integrate them with Velo API's to ensure accurate data management and enhanced customization. Each app is identified by a unique `appId`.
```

Comment:
I think you should put in your link with fake placeholder text already, so I can see how you intend to mention the link.

Reply:
This sounds very negative: "to ensure accurate data management" ... it sounds like if you don't use these business solutions or you don't use Velo,  you cannot trust that your data is accurate.  

Also, no need to reinvent the wheel.  Just copy Rebecca's 1st paragraph "as is."  I see some issues with her first paragraph also, but at least we will be consistent.

Reply:
Then I would add a 2nd paragraph that starts with your sentence  (but obviously we have to update the actual dummy link I put there for data quotas, as I know you know):  

Each app is identified by a unique `appId`. See a complete list of Made by Wix app IDs [here](https://dev.wix.com/docs/develop-websites/articles/databases/about-data-quotas).

Reply:
I'm very confused about what you want here. 
1. placeholder text: I have added placeholder text [Apps made by Wix].
2. The last time we met, you said that you didn't like Rebecca's article, and that it was only a half truth that apps are used to manage data.